laborator/content/operatii-memorie-gdb/1-iterate/Makefile

```diff
@@ -1,2 +1,2 @@
-PROGNAME := delete-first
+PROGNAME := iterate
```

Comment:
Use the imperative mood in the commit subject. Also, add  small prefix showing the section where this change applies.

e.g `lab: Fix naming of compiled files`